committer/src/setup.rs

```diff
@@ -134,6 +123,8 @@ pub fn state_committer(
             lookback_window: config.app.bundle.block_height_lookback,
             fragment_accumulation_timeout: config.app.bundle.fragment_accumulation_timeout,
             fragments_to_accumulate: config.app.bundle.fragments_to_accumulate,
+            gas_bump_timeout: Duration::from_secs(300), //TODO: make this configurable
```

Comment:
Do you have any thoughts on doing that now? This could be something we would like to quickly control in the deployed environments.